### ISSUE TITLE
The Expanse - Cropped View

### ISSUE THREAD

**Issue by Azulath** [RoseTheFlower/UltrawideIndex#56]
It seems like the last two episodes the view is cropped to fit the screen. i.e., both the very bottom and the very top are cut off.

**Comment by RoseTheFlower** [comment]
Could you share your save files from Documents\My Games\Telltale\Expanse\Saved?

A screenshot of what you perceive as cropping would help as well.

**Comment by RoseTheFlower** [comment]
Turns out the relevant code is somewhat different in the latest game version, so I adapted the fix to it and released as v1.2.

**Comment by RoseTheFlower** [comment]
Right, no need unless you run into the issue again with the latest version.
